extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -25,45 +29,86 @@ export class SKURecommendationPage extends MigrationWizardPage {
 	private igComponent: azdata.FormComponent<azdata.TextComponent> | undefined;
 	private detailsComponent: azdata.FormComponent<azdata.TextComponent> | undefined;
 	private chooseTargetComponent: azdata.FormComponent<azdata.DivContainer> | undefined;
+	private azureSubscriptionText: azdata.FormComponent<azdata.TextComponent> | undefined;
```

Comment:
we should be consistent on use of the '_' prefix for private fields within a class.